Intersect.Server/Entities/Entity.cs

```diff
@@ -233,6 +231,9 @@ public string FooterLabelJson
 
         [NotMapped, JsonIgnore]
         public long MoveTimer { get; set; }
+        
+        [NotMapped, JsonIgnore]
+        private readonly float PythagoreanMultiplier = (float) Math.Sqrt(2);
```

Comment:
This should be a static, and since this constant exists in another file, please instead move UnityDiagonalLength to live in MathHelper (I think that is in Core? if it isn't in core add a MathConstants static class)

Reply:
Intersect (Core)/Utilities/MathHelper.cs

![imagen](https://user-images.githubusercontent.com/17498701/222857341-36a91208-7278-435f-9a07-5a603dcec7ae.png)

![imagen](https://user-images.githubusercontent.com/17498701/222857362-6ae77a9e-c23d-43c8-a9fd-b6f9b1413350.png)